libs/books/feature/src/lib/book-search/book-search.component.ts

```diff
@@ -34,6 +35,13 @@ export class BookSearchComponent implements OnInit {
 
   ngOnInit(): void {
     this.books = this.store.select(getAllBooks);
+
+    this.searchForm
```

Comment:
No need to emit if the term stays the same (rapidly hit a character, then backspace, for instance)